database/migrations/2018_04_16_162649_basic_tables.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class BasicTables extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('workflows', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger("authorId");
+            $table->timestamp("created")->useCurrent();
```

Comment:
Why not use the built-in functionality `$table->timestamps()`?

Reply:
I wanted to set a default value for the "created" attribute.

Reply:
I have just seen in the documentation that setting creation date is done automatically by Laravel, however I believe it is not a good practice to change previous migrations, I'll just update the names in a new migration.

Reply:
Yes, you are correct. You should never change a migration that was already pushed to `development`